index.html

```diff
@@ -12,7 +12,21 @@
 </head>
 
 <body>
+
     <div class="container">
+        <div class="logo">
+            <img src="img/lambda-black.png" alt="black lambda logo">
+       </div>
+        <nav>
+            <a href="#">Home</a>
+            <a href="about.html">About</a>
```

Comment:
great job adding the navbar and linking to the about.html page. you keep your code clean and properly indented.